modules/apps/bean-portlet/bean-portlet-cdi-extension/src/main/java/com/liferay/bean/portlet/cdi/extension/internal/BeanPortletExtension.java

```diff
@@ -273,6 +276,35 @@ public void step1BeforeBeanDiscovery(
 		}
 	}
 
+	public <T> void step2ProcessInjectionTarget(
```

Comment:
@dantewang It's fine to rename this method `step3ProcessInjectionTarget` and then renumber the step3, step4, etc.. method names accordingly.